CONTRIBUTING.md

```diff
@@ -0,0 +1,95 @@
+## Set up your development environment
+
+You'll need a Phantom OVA VM image to run this application. To download the image, first register [here](https://my.phantom.us/signup/)
+to create a Phantom account. Phantom will approve your registration and send you credentials for https://my.phantom.us.
+
+Next, log in and go [here](https://my.phantom.us/downloads/) to download the Phantom VM image.
+
+Follow these [instructions](https://docs.splunk.com/Documentation/Phantom/4.10.3/Install/InstallOVA) to install Splunk Phantom 
+as a VM image. You do NOT need to complete the section labeled "Configure the network settings for the virtual machine".
+
+Use these [default credentials](https://docs.splunk.com/Documentation/Phantom/4.10.3/Install/Reference) to log in.
+Confirm that you can connect to the VM via SSH on your host machine. You can use `hostname -I` on the remote machine to look up your IP address.
+Also confirm that you can access the Phantom web app at `https://{phantom VM IP address}`.
```

Comment:
The spaces in `phantom VM IP address` cause a weird underline thing in VSCode, could we put dashes or something in between the words?